mealy/preprocessing.py

```diff
@@ -160,7 +160,7 @@ def transform(self, x):
         """Transform the input feature values according to the preprocessing pipeline.
 
         Args:
-            x (numpy.ndarray or pandas.DataFrame): input feature values.
+            x ({array-like, dataframe} of shape (n_samples, n_features)): input feature values.
```

Comment:
Extreme nitpitck
```suggestion
            x (array-like or dataframe of shape (number of samples, number of features)): input feature values.
```
Same on line 309. Otherwise, LGTM